go/runtime/host/sgx/ecdsa.go

```diff
@@ -184,7 +187,13 @@ func (ec *teeStateECDSA) Update(ctx context.Context, sp *sgxProvisioner, conn pr
 		}
 		ec.tcbCache.cache(fresh, pckInfo.FMSPC)
 		return fresh, nil
-	}()
+	}
+	var tcbBundle *pcs.TCBBundle
+	for _, update := range []pcs.UpdateType{pcs.UpdateStandard, pcs.UpdateEarly} {
+		if tcbBundle, err = getTcbBundle(update); err == nil {
```

Comment:
I would log error here, to log also errors from line 174. Or maybe warn standard and error early.